skill_library/Affordance affordance
---
skill_library/Affordance affordance
int32 result
int32 SUCCEEDED=1
int32 FAILED=2
